### ISSUE TITLE
NoWildcardImports excludeImports

### ISSUE THREAD

**Issue by jonas-arkulpa** [arturbosch/detekt#1940]
As the conversation on the related issue https://github.com/arturbosch/detekt/issues/1468 is locked I arrogate to create a new one. 😉 

We currently have the same issue with `java.util.*` imports.

I understand the difference between `WildcardImport` and `NoWildcardImports` (ktlint).

With the workaround described in https://github.com/arturbosch/detekt/issues/1468#issuecomment-476390151 I would loose the autoCorrect feature.

Is it possible to workaround this limitation?

## Expected Behavior
Excludes and auto corrections can be used simultaniously.

## Observed Behavior
These ktlint and detekt features can't be properly combined.

![NoWildcardImportsErrorExample](https://user-images.githubusercontent.com/17269256/65317263-cc915080-db9b-11e9-96d6-a825184799fc.png)

## Steps to Reproduce
```yaml
formatting:
  active: true
  android: true
  autoCorrect: true
  NoWildcardImports:
    active: true
    autoCorrect: true

style:
  active: true
  WildcardImport:
    active: true
    excludes: "**/test/**,**/androidTest/**,**/*.Test.kt,**/*.Spec.kt,**/*.Spek.kt"
    excludeImports: 'java.util.*,kotlinx.android.synthetic.*'
```

**Comment by cortinico** [comment]
> I would loose the autoCorrect feature.

Please note that `NoWildcardImports` doesn't have the `autoCorrect` support. 
That being said, I think we can close this issue?

**Comment by jonas-arkulpa** [comment]
@cortinico You are right. I'm so dumb, noted the same thing a couple months ago. 🙈 

https://github.com/arturbosch/detekt/issues/1746#issuecomment-507797897
> Also found out that ktlints NoWildcardImportsRule doesn't support auto correction. The autoCorrect: true config in the detekt config was kind of misleading.
